thirdeye/thirdeye-pinot/pom.xml

```diff
@@ -239,6 +239,12 @@
       <artifactId>parsii</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
we should not bring this dependency here. can we make this a runtime dependency

Reply:
@kishoreg, what is the rationale behind this?

Reply:
Its a bit pre-mature but not abstracting these right now will result in a classpath nightmare. We are seeing this in Pinot and I feel its better to break optional stuff into interface and implementation. And then move the implementation to a separate module and load them at runtime. Once we have this, it makes it easy to add multiple implementations. For e.g. we might use some other system for caching.